src/Input/Input.less

```diff
@@ -1,5 +1,8 @@
 /* Input.css */
 .moon-input {
+	line-height: 57px;
```

Comment:
I think this 'line-height' can override value from 'moon-body-text'.

Reply:
It will depend on the precedence of the rules and the order of the classes, since this is a "top-level" class, just like `moon-body-text`. This code will need to fall after the selectors in `text.less`